db/schema.rb

```diff
@@ -535,7 +557,7 @@
     t.datetime "updated_at", null: false
     t.string "last_prompted_upgrade_commit_sha"
     t.string "last_alerted_upgrade_commit_sha"
-    t.string "role", default: "member", null: false
+    t.enum "role", default: "member", null: false, enum_type: "user_role"
```

Comment:
This appears to be from the recent impersonations migration.  I think you'll need to run a `db:reset` to clear this out since this _should_ be a `t.string` now.

Reply:
I had to add a new migration to just fix it for good. I think the problem from before was that the migration file from the original impersonations was editing after it'd already been run on my end. So we just ended up with a jumbled mess.

Running `db:reset` didn't fix it unfortunately.

_Should_ be addressed now, though.